ecosystem/resources/members/quPython.toml

```diff
@@ -1,6 +1,6 @@
 name = "quPython"
 url = "https://github.com/frankharkins/quPython"
-description = "Write quantum programs as Python functions, rather than separate circuit objects. Create higher-level quantum data types, and return measurement results as bool-like objects."
+description = "Write quantum programs as Python functions rather than circuit objects. Create higher-level quantum data types and return measurement results as bool-like objects."
```

Comment:
```suggestion
description = "Write quantum programs as Python functions instead of circuit objects. Create higher-level quantum data types and return measurement results as bool-like objects."
```